Gemfile

```diff
@@ -24,7 +24,7 @@ gem 'aws-sdk-ses', '~> 1'
 gem 'barnes' # heroku ruby-specific metrics
 gem 'bootstrap', '~> 5.3' # pin until major version is handled
 gem 'dartsass-sprockets'
-gem 'exception_notification'
+gem 'exception_notification', git: 'https://github.com/Throne3d/exception_notification.git', ref: 'b2131cf'
```

Comment:
Fixes frozen_string_literal support (https://github.com/smartinez87/exception_notification/pull/549) and relaxes the `rails < 8` constraint (https://github.com/smartinez87/exception_notification/pull/544)